joomla/joomla-cms
# add flip ordering batch process to category manager

Pull Request for Issue #11103 .

This is not a real fix for the issue, but it is a way to make the B/C break not a big pain. In combination #11475 it allows sites that are using ordering to invert the ordering of articles with one click. 

#### Summary of Changes
It adds a yes/no select to the batch process in the category manager to invert the ordering within categories. 

#### Testing Instructions
* Apply patch
* You need a category view with articles
* Set the the secondary ordering in Articles options to „Ordering Reverse“
* Set the the secondary ordering for the menu to „Use Global“
* Is is easier when you limit the view to one category and set leading 0, intro 20, columns 1, links 0 
* check the view and memorise the ordering
* go to backend, select the category in the category manager and click on batch.
* select yes under "Reverse the ordering of selected categories"
* process
* go to the frontend, the ordering should now inverted

Note: I have tested this with a category with 30000 articles and there was only a small delay.

#### Documentation Changes Required
Because this is a "fix" for a B/C break we should put at a prominent place in the release notes.
